templates/config/controller/deployment.yaml.tpl

```diff
@@ -49,3 +51,18 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
       terminationGracePeriodSeconds: 10
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ack-{{ .ServiceIDClean }}-metrics-service
+  namespace: ack-system
+spec:
+  selector:
+    control-plane: controller
+  ports:
+    - name: metricsport
+      port: 8080
+      targetPort: 8080
+      protocol: TCP
+  type: NodePort
```

Comment:
Would be great to put this into a separate `service.yaml.tpl` file in a future PR...

Reply:
created `service.yaml.tpl` file.